src/codegen/codegen_cpp_visitor.cpp

```diff
@@ -166,6 +166,13 @@ bool CodegenCppVisitor::defined_method(const std::string& name) const {
     return function && function->has_any_property(properties);
 }
 
+bool CodegenCppVisitor::is_function_table_call(const std::string& name) const {
+    auto it = std::find_if(info.function_tables.begin(),
```

Comment:
`std::any_of` would also work here (saving a grand total of 1 line 😄)

Reply:
Thank you, there's plenty more in other parts. We definitely need something to tame all the `std::find`s.